react/features/base/participants/middleware.js

```diff
@@ -89,13 +91,17 @@ MiddlewareRegistry.register(store => next => action => {
             break;
         }
 
-        participant
-            && store.dispatch(participantUpdated({
-                conference,
-                id,
-                local: isLocal,
-                raisedHand: false
-            }));
+        const updatedData: Object = {
+            conference,
+            id,
+            local: isLocal
+        };
+
+        if (!getDisableRemoveRaisedHandOnFocus(state)) {
+            updatedData.raisedHand = false;
+        }
+
+        participant && store.dispatch(participantUpdated(updatedData));
```

Comment:
You can avoid dispatching that action altogether, if we are not updating the raisedHand value.

Reply:
Yes, you are correct, thank you, it has been resolved